frontend/src/__tests__/cypress/cypress/pages/projects.ts

```diff
@@ -169,7 +169,7 @@ class ProjectDetails {
     );
   }
 
-  findImportPipelineButton(timeout?: undefined) {
+  findImportPipelineButton(timeout?: number) {
```

Comment:
We really need to have a TS eval happen here @christianvogt this was invalid TS submitted by a previous PR.